adcio_core/src/main/java/com/corcaai/adcio_core/feature/AdcioCore.kt

```diff
@@ -0,0 +1,63 @@
+package com.corcaai.adcio_core.feature
+
+import android.os.Build
+import com.corcaai.adcio_core.error.NotInitializedException
+import java.util.UUID
+
+object AdcioCore {
+
+    private var isInitialized: Boolean = false
+
+    fun init(id: String) {
```

Comment:
init 명칭 통일화 할게요~ firebase_core에서 initializeApp로 초기화 하니 우리도 비슷하게 가겠습니다!
```suggestion
    fun initializeApp(id: String) {
```